lib/migrate_ast.ml

```diff
@@ -32,7 +32,7 @@ module Parsetree_ = Parsetree
 
 include (
   Migrate_parsetree.Ast_406 :
-  module type of struct
+    module type of struct
```

Comment:
why did this file changes? can you clean-up the history and remove the merge commit?

Reply:
Indeed, I was late with master, that's why, i'll do the history rework